aces/Loris
# [Candidate List] change visitLabel filter to dropdown, fix link issue

Re-issue of #5362 

### Brief summary of changes
This PR does 3 things:
1. It changes the visit label filter from a textbox to a dropdown. 
2. It allows for proper filtering of subproject and visit label. Using exactMatch within the array.
3. It addresses the issue reported in #5360 in which if a user does not have `across all sites` permission, links to the candidate profiles do not appear. However, the list of candidates returned in the table is already based on their permissions and only displays the candidates that they SHOULD have access to. 

#### Testing instructions (if applicable)
1. Try filtering by visit label and / or subbproject
2. Remove the `across all site` permission and see if you are able to access profiles. 

#### Links to related tickets (GitHub, Redmine, ...)
* #5360
* #5359
